pkg/authenticator/authenticator.go

```diff
@@ -133,7 +152,35 @@ func (auth *Authenticator) Login() error {
 		return log.RecordedError(log.CAKC029E, err.Error())
 	}
 
-	// load client cert
+	return nil
+}
+
+// Returns true if we are logged in (have a cert)
+func (auth *Authenticator) IsLoggedIn() bool {
+	return auth.PublicCert != nil
```

Comment:
again, couldn't this present a problem if we have a cert but it is no longer valid?